libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -99,24 +99,26 @@ - (id)contentFromData:(NSData*)data fromResponse:(NSHTTPURLResponse*)response er
     NSError *responseError = nil;
     id content = [super contentFromData:data fromResponse:response error:&responseError];
 
-    if (content && !responseError) {
+    if (!responseError) {
```

Comment:
In the case of a 404 (which is what the test does), we have a nil content and no responseError. If we don't enter the if, the code will get back a nil object and no error.